superset/config.py

```diff
@@ -172,6 +172,9 @@
 ENABLE_CORS = False
 CORS_OPTIONS = {}
 
+# CSV Options
+CSV_SEP = ','
```

Comment:
Since CSV has a ton of options and pandas does not support dialects i think a CSV_EXPORT dict would be more future proof:

```
CSV_EXPORT = {
    'SEPARATOR': ',',
    'ENCODING': 'utf-8',
}
```

Reply:
Indeed, this looks better. I will implement it this way.

Reply:
Actually, I was thinking I can do something more generic to be able to replace any option of the to_csv method. I'm gonna change this and resubmit.